src/widgets/canvas.rs

```diff
@@ -146,27 +196,25 @@ impl Grid for CharGrid {
     }
 
     fn resolution(&self) -> (f64, f64) {
-        (f64::from(self.width) - 1.0, f64::from(self.height) - 1.0)
+        (f64::from(self.width), f64::from(self.height))
     }
```

Comment:
Does this need a `breaking change` label?

Reply:
No.

The grid trait is private, and the grids themselves are also, so nothing of this is actually user facing. 

I just moved the -1 outside the resolution calculation because I have a 3840x2160 4K monitor, not a 3839x2159 maximum pixel index monitor ;) I should probably document it as a note in the commit however.

Reply:
oops, didn't notice it was not `pub fn`!